setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
+name = stonesoup
+description = A tracking and state estimation framework
+long_description = file: README.md
+long_description_content_type = text/markdown
+maintainer = Defence Science and Technology Laboratory UK
+maintainer_email = oss@dstl.gov.uk
+url = https://github.com/dstl/Stone-Soup
+license = MIT
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Scientific/Engineering
+
+[options]
+python_requires = >=3.7
+packages = find:
+install_requires =
+    matplotlib
+    numpy>=1.17
+    ordered-set
+    pymap3d
+    ruamel.yaml>=0.16.5
+    rtree
+    scipy
+    utm
+
+[options.extras_require]
+dev =
```

Comment:
With #707, we will also need pandas here. But maybe that should be addressed in that PR.

Reply:
Yeah, that PR adds the dependency. This'll cause a merge conflict, so on merge of 707 I'll add pandas onto this list.